code/game/jobs/job/silicon.dm

```diff
@@ -24,8 +24,8 @@
 	title = "Cyborg"
 	flag = JOB_CYBORG
 	department_flag = JOBCAT_ENGSEC
-	total_positions = 1
-	spawn_positions = 1
+	total_positions = 2
+	spawn_positions = 2
```

Comment:
I'm inclined to say if you're going to do this, there should only ever be *ONE* security borg active at a time.

Reply:
for the population on the server we have, i think 1 is not enough.

Reply:
One is definitely not enough.

Reply:
You'd be surprised---a lot of AI's are pretty validy and specifically *order* their borgs to go security.

Objectively speaking, on average, there's going to be *more* sec borgs running around with this change than before.

Even if only 1 in 4 rounds the second borg goes security borg, that's still, on average, more than current.

Reply:
I've adjusted the limit on secborgs down to 1 at roundstart, and on green alert. 2 on higher security alert levels. This will stop multiple roundstart cyborgs going secborg. I'm not willing to set the cap on secborgs to 1 at all times, even on red alert. I feel that goes too far. I hope this compromise will be enough.